### ISSUE TITLE
Labels for the dataset

### ISSUE THREAD

**Issue by hellorp1990** [AleGiovanardi/covidhelper#1]
Hi,

Its an excellent work. I am little confused with the test folder as it has a test folder inside it along with some .jpeg images.  Can you please clarify this?

Another thing is the test folder (cttest) inside the CT data, It doesn't have any labels. I know its a work in progress. But if you can give some more details about the dataset, it will be really great.

How did you choose an image from the CT scan for a patient ?

For the CT scans can you please provide the whole scan series?

Thanks.

**Comment by AleGiovanardi** [comment]
Hi, thanks for your support. Folder structure in the commit you are commenting about is WIP yet and I cleared it a bit in the next commit I'm going to push soon.

I am still collecting data to make a decent training dataset both for RX and CT scans. 

CT scans comes from publications found all over the internet plus many originals coming from hospitals in my area.

I spoke with a medic and he affirmed that you choose the image which better represents the visualization of the pathology you are looking for.

I have full CT sequence of a few files only from which i took frames to test against the trained model. I used them in my last test (and will be uploaded in the next commit).

Feel free to contribute or help the project in anyway.
